docs/faq-and-others/faq.md

```diff
@@ -70,3 +70,11 @@ You can use the [`COPY TO` command](https://docs.greptime.com/reference/sql/copy
 ## TSDB features that you concern
 
 Please refer to [features that you concern](/user-guide/concepts/features-that-you-concern.md).
+
+## Does GreptimeDB support auto-increment primary keys similar to MySQL?
+
+Not yet.
```

Comment:
Will it be supported in the future?

Reply:
Jiachun says now we don't have the plan to support it since time-series scenario is not common

Reply:
a little advice: explain further why a time-series database does not require auto-increment, in order to help users better understand time-series usage scenarios.

Reply:
I suggest deleting this question as I believe that auto-increment ID is more appropriate for TP database scenarios and not commonly used in time series.